src/main/java/hobbylist/logic/parser/ParserUtil.java

```diff
@@ -106,6 +107,28 @@ public static List<Date> parseDate(List<String> listDate) throws ParseException
         return s;
     }
 
+    // Solution adapted from https://github.com/AY2021S1-CS2103T-F11-3/tp/pull/124/files
+
+    /**
+     * Parses a {@code String status} into a {@code Status}
+     *
+     * @throws ParseException
+     */
+    public static Status parseStatus(String status) throws ParseException {
```

Comment:
If this takes an Optional rather than a String you can drop the check mentioned above.